bin/cli.js

```diff
@@ -40,11 +43,29 @@ function apply(dbName, schemaName, cmd) {
         : db.apply(options);
       return vow.finally(() => db.disconnect());
     })
-    .catch(err => console.log(err.messsage));
+    .catch(err => {
+      console.error(err.messsage);
+      process.exit(1);
+    });
+}
+
+function getLatestOrdinal(dbName, schemaName, cmd) {
```

Comment:
The new command line program to print out the latest version of the migration to stdout